app/templates/CommentList.rss.twig

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0">
+    <channel>
+        <title>Spenden an Wikimedia</title>
+        <link>{$ rssLink |e $}</link>
```

Comment:
You sure this is on for this template? In case of security, I much prefer explicitness over smart magic to reduce verbosity.

In any case, should also add a test to verify stuff is being escaped